modules/db/src/test/java/org/mule/module/db/integration/model/MySqlTestDatabase.java

```diff
@@ -142,6 +142,21 @@ public void createStoredProcedureConcatenateStrings(DataSource dataSource) throw
     }
 
     @Override
+    public void createDelayFunction(DataSource dataSource) throws SQLException
+    {
+        executeDdl(dataSource, "DROP FUNCTION IF EXISTS DELAY;\n");
+
+        final String sql =
+                "" +
```

Comment:
Can this empty line be removed so the formatting is consistent with OracleTestDatabase?